util/base/src/test/java/jetbrains/jetpad/base/edt/TestEventDispatchThreadTest.java

```diff
@@ -89,4 +94,32 @@ public void run() {
     });
     edt.executeUpdates();
   }
+
+  @Test
+  public void exceptionInTask() {
```

Comment:
we now need two tests for edt.executeUpdates() call - for a plain one and for one wrapped with ThrowableHandlers.asInProduction()